src/metaswitch/ellis/prov_tools/utils.py

```diff
@@ -236,3 +244,48 @@ def display_user(private_id, public_id, short=False):
             success = False
 
     return success
+
+
+def list_users(target_users_per_chunk=100, keep_going=False):
```

Comment:
I understand what the algorithm is here because you told me, but a function comment ("Query the targeted amount of users per second by querying as few as possible, and then requesting twice as many on each successive query until we're about right"?) might be useful here.